README.md

```diff
@@ -0,0 +1,11 @@
+# Description
+
+This is a clone of a New York Times article page that features the following conceptss
+-Embedding images
```

Comment:
#### `HINT`

Add spaces to make a markdown list. Kindly, do this for other list items.

```suggestion
- Embedding images
```